microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -772,6 +772,17 @@ Microsoft Edge WebView controls allow app developers to host web content in nati
 
 To enable diagnostic data collection, go to `edge://settings/privacy`. Turn on the **Optional diagnostic data** setting. To turn off diagnostic data collection for Microsoft Edge on Windows 10 and later, open **Start** > **Settings** > **Privacy** and select **Diagnostics & feedback**. To turn off diagnostic data collection for all other platforms, go to `edge://settings/privacy`. Turn off the **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports** setting. The applications hosting Microsoft Edge WebView may collect other data that is governed by the data collection management of the developer and relevant privacy policies.
 
+
+<!-- ====================================================================== -->
+## Workspaces
+
+The Microsoft Edge Workspaces feature allows you to easily organize and share your browsing tasks with collaborators via customizable, shareable browsing windows. Each workspace shares a set of tabs, favorites, and history, all created and curated by you and your collaborators. Workspace data is automatically saved, kept up-to-date, and stored in OneDrive/SharePoint.
```

Comment:
```suggestion
The Microsoft Edge Workspaces feature allows you to easily organize and share your browsing tasks with collaborators via customizable, shareable browsing windows. Each workspace shares favorites, a set of workspace tabs, and history, all created and curated by you and your collaborators. Workspace data is automatically saved, kept up-to-date, and stored in OneDrive/SharePoint.
```
i found phrasing confusing since i don't know this feature:
a set of a, b, and c -- is the set [a, b, and c]?  or, [a set of a] and also b and c?
Instead, trying phrase from below, "workspace tab".  And different order so it's clear that "set of" only applies to the tabs - if that's the intention.

Or maybe the intention is:
Each workspace shares a set of [tabs, favorites, and history].

Reply:
The intention is "Each workspace shares a set of [tabs, favorites, and history]." I do see where that phrasing could be confusing, though. How does the following sound, with the removal of "set of"? 

Each workspace shares tabs, favorites, and history.